### DISCUSSION TITLE
Grott Docker update

### DISCUSSION THREAD

**Discussion by mvank** [johanmeijer/grott#55] [General]
Hi @johanmeijer,

when will the [docker version of grott](https://hub.docker.com/r/ledidobe/grott) be updated to version 2.6 ?

**Comment by johanmeijer** [comment]
It will do it when I am back witrhin the "normal" IT environment (>19th july). I will make a reminder in my Agenda of it.

Do you use the RPI or x86 version ?

**Reply by mvank** [reply]
Thank you @johanmeijer 
I am using x86, running docker on a NUC combined with Proxmox.

**Comment by johanmeijer** [comment]
@mvank 

I created the 2.6.1 docker image. For know you can start it by using the 2.6.1 label (so ledidobe/grott:2.6.1). 

At some point this week I will it also so make it the latest version!

**Comment by johanmeijer** [comment]
@mvank have you already tried the new docker image?

**Comment by mvank** [comment]
Hi @johanmeijer, sorry did not see the beta version.
Just updated to beta version 2.6.1
As far as I can see, growatt portal shows updates and also home assistant via MQTT.
Still working as expected, will see tomorrow what the solar panels will do? I hope for better weather! ;-)

**Comment by mvank** [comment]
Hi @johanmeijer, new docker release is working fine, have not seen any issue.

**Comment by johanmeijer** [comment]
Thanks for testing. I will promote him to the latest version.

**Comment by ZonPV** [comment]
I run the container of ledidobe-grott version 2.5.4. in docker on my Synology. Not the latest one. Version 2.6.1 is available..

Because of the different name of the container I was searching in docker for JohanMeijer :-) I found that name but could not download that image. It says: there is no label... 
[<a href="https://imgbb.com/"><img src="https://i.ibb.co/8s7Nqrf/Image-heeft-geen-tag.jpg" alt="Image-heeft-geen-tag" border="0"></a><br /><a target='_blank' href='https://nl.imgbb.com/'>photo free image</a><br />](url)
So I found an update for the image I use at the moment:

[<a href="https://ibb.co/x2HPMFv"><img src="https://i.ibb.co/8mcTjY3/Download-Grott.jpg" alt="Download-Grott" border="0"></a><br /><a target='_blank' href='https://nl.imgbb.com/'>photo free image</a><br />](url)
Wondering if these versions have the same content and author but only different names?

**Comment by ZonPV** [comment]
Found out that Ledidobe is his pseudonym! Thank you for spreading your knowledge!

**Reply by johanmeijer** [reply]
Yep the ledidobe grott images on docker are the original ones ;)

**Reply by ZonPV** [reply]
Good to know. Thanks for your confirmation!
